packages/edit-site/src/components/post-fields/index.js

```diff
@@ -199,6 +199,16 @@ function usePostFields( viewType ) {
 				),
 				enableSorting: false,
 			},
+			{
```

Comment:
Do we want/expect "Pinned" to be a new field that users can toggle on/off? If we don't, we shouldn't need this to make the feature work, I presume?